src/sentry/static/sentry/app/actionCreators/modal.tsx

```diff
@@ -120,7 +120,7 @@ export async function openCommandPalette(options: ModalOptions = {}) {
   );
   const {default: Modal, modalCss} = mod;
 
-  openModal(deps => <Modal Body={deps.Body} {...options} />, {modalCss});
+  openModal(deps => <Modal {...deps} {...options} />, {modalCss});
```

Comment:
Nit: It wasn't immediately clear to me that `dep == props`. Would it be better to rename `deps` to `props` for all other functions?

Reply:
Yeah it's a little weird, we should probably rename the argument in all usages of `openModal`